docs/en_US/Compressor/Quantizer.md

```diff
@@ -6,10 +6,12 @@ We provide Naive Quantizer to quantizer weight to default 8 bits, you can use it
 
 ### Usage
 tensorflow
-```python nni.compression.tensorflow.NaiveQuantizer(model_graph).compress()
+```
+python nni.compression.tensorflow.NaiveQuantizer(model_graph).compress()
```

Comment:
tensorflow section should be removed for now.

pytorch
```
```python
model = nni.compression.torch.NaiveQuantizer(model).compress()
```

Reply:
move the `python` up one line next to the quotes